homeassistant/components/ihc/ihcdevice.py

```diff
@@ -67,7 +69,7 @@ def name(self):
     @property
     def unique_id(self):
         """Return a unique ID."""
-        return f"ihc{self.controller_id}{self.ihc_id}"
+        return f"{self.controller_id}{self.ihc_id}"
```

Comment:
Consider adding some separator eg `-`, that is not present in the ids, between the controller id and ihc id to allow parsing the unique_id and get the parts more easily in the future.

Reply:
That is a good idea. I don't know 100% how the controller serial is formatted on the different models, or will be formatted in future models. I have an older ihc controller where it is just digits, a newer on where it is a combination of letters and digits. The ihc_id is always a number, so if I use - it should work. In case - later will be a part of the serial, it will be possible to search for the last - because the ihc_id is always digits. 
I have added - as separator